posthog/caching/tolerant_zlib_compressor.py

```diff
@@ -30,6 +41,8 @@ def decompress(self, value: bytes) -> bytes:
         try:
             return zlib.decompress(value)
         except zlib.error:
+            if settings.USE_REDIS_COMPRESSION:
```

Comment:
Is the conditional relevant? Presumably if that error has been thrown then we are using compression or at least trying to?

Reply:
We read through this compressor regardless of setting, so in prod with the setting off we are logging a warn for every cache get even though we've never compressed anything 🙈